packages/css/src/plugins/parent-orphaned-pseudos.tsx

```diff
@@ -32,7 +32,7 @@ export const parentOrphanedPseudos = plugin('parent-orphened-pseudos', () => {
     root.walkRules((rule) => {
       const { selector: ruleSelector } = rule;
 
-      if (!ruleSelector.includes(':')) {
+      if (!ruleSelector.startsWith(':')) {
```

Comment:
this is technically a regression but because we have the nesting logic bled into the atomicify plugin it doesnt break

my follow up PR fixes this